web/app/features/Blog/components/Card/Card.tsx

```diff
@@ -43,10 +43,11 @@ export function Card({ post, excerpt, picture, small = false }: CardProps) {
           href={route('/:slug', {
             slug: slug.current,
           })}
+          data-testid="post-link"
```

Comment:
**dev note**: this is the only relevant change. Everything else is just Tailwind ESLint doing its job.